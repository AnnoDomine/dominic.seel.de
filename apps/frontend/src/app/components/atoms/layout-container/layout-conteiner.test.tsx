import { it, describe, expect } from "vitest"
import { render, screen } from "@testing-library/react"
import LayoutContainer from "./layout-container"

describe("LayoutContainer", () => {
    it("renders without crashing", () => {
        render(<LayoutContainer data-testid="layout-container">Content</LayoutContainer>)
        const layoutContainer = screen.getByTestId("layout-container")
        expect(layoutContainer).toMatchSnapshot()
    })
})
