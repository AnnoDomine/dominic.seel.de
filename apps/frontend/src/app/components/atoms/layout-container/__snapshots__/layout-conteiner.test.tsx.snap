// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LayoutContainer > renders without crashing 1`] = `
<div
  class="css-1c5db69"
  data-testid="layout-container"
>
  Content
</div>
`;
